pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -169,9 +169,9 @@ type SourceObjectKeySelector struct {
 	//+optional
 	Selector *metav1.LabelSelector `json:"selector,omitempty"`
 
-	// KeySelector is the key of the entry in the objects' `data` field to be referenced.
+	// Key is the key of the entry in the objects' `data` field to be referenced.
```

Comment:
```suggestion
	// Key of the entry in the object's `data` field to be used.
```

Possible better aligned with the new `IncludeAllKeys` field?

Reply:
Perfect, done! I've squased all commits into one below: https://github.com/cert-manager/trust-manager/pull/474/commits/18ba82c5d77f26cc3a8675d18ddd4121fee657d6